pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.mypy]
+warn_unused_configs = true
+python_version = "3.9"
+
+[[tool.mypy.overrides]]
+module = ["pyodide", "pyodide_js", "pyodide.http", "pytest", "ruamel.yaml"]
+ignore_missing_imports = true
```

Comment:
Do you think it's better bo restrict this to those modules as opposed to everything?

Reply:
Yes, for normal code, it helps find bugs - ignoring an import hides bugs. If you can get the import, that's much better.

If you are referring to `pyodide", "pyodide.http"` vs. `"pyodide.*"`, I just went with minimal over blanket, have no idea what's better. Or if we really should be ignoring these, ideally they are typed and can be found. Does pyodide have a `py.typed` file?

Reply:
Your proposed setup sounds good to me, I was just wondering aloud as have never used that setting before.

> Does pyodide have a py.typed file?

No, but we probably should.

Reply:
Possibly dumb question: where is `pyodide`? I see `/src`, but not `/src/pyodide`. Probably why mypy can't find it either.

If mypy does find a package, it uses it - ignore_missing_imports just means it won't tell you if it doesn't find it, and makes it Any instead.

Reply:
It's at src/py/pyodide. We have the python code under src/py, the typescript code which isn't run through the C preprocessor in src/js and a mix of C as well as typescript and javascript which get run through the C preprocessor in src/core.

Reply:
Perfect, fixed this locally. Will update in the next push (after I know what to do with the missing imports).